gradle/libs.versions.toml

```diff
@@ -10,17 +10,17 @@ opentelemetryProto = "1.3.2-alpha"
 
 # otel agent, we rely on the '*-alpha' and get the non-alpha dependencies transitively
 # updated from upstream agent with .ci/update-upstream.sh
-opentelemetryJavaagentAlpha = "2.12.0-alpha"
+opentelemetryJavaagentAlpha = "2.13.3-alpha"
 
 # otel contrib
 # updated from upstream agent with .ci/update-upstream.sh
-opentelemetryContribAlpha = "1.42.0-alpha"
+opentelemetryContribAlpha = "1.44.0-alpha"
 
 # otel semconv
 # updated from upstream agent with .ci/update-upstream.sh
 # While the semconv stable artifact is provided transitively by the agent, we still have to explicitly
 # reference the "incubating" version explicitly
-opentelemetrySemconvAlpha = "1.29.0-alpha"
+opentelemetrySemconvAlpha = "1.30.0-rc.1"
```

Comment:
This will also require a change to our gradle task printing the release notes, which currently has the `-alpha` suffix hardcoded. We should probably replace that with a `-.*` regex.
https://github.com/elastic/elastic-otel-java/blob/main/build.gradle.kts#L57

Reply:
done in e950781fdd93d5d52a7d0e9192fa521628bdb914